### ISSUE TITLE
rio attach now showing pod stdout

### ISSUE THREAD

**Issue by izaac** [rancher/rio#608]
**Describe the bug**
doing a `rio attach <service>` is now showing the container stdout as in 0.5.0

**To Reproduce**
1. Deploy a new service `rio run --name attachtest izaac/attachtest:v1`
2. `rio attach attachtest`

Nothing is shown after the service is ready, this container shows the `date` every 5 seconds to stdout.

**Expected behavior**

Attach Should show output from the pod in this case output of the running bash script running `date` every 5 seconds.

**Kubernetes version & type (GKE, on-prem)**: `kubectl version`
```
v1.14.7-gke.10
```
Type:
**Rio version**: `rio info`
```
Rio Version: v0.6.0-alpha.1 (7678d67d)
Rio CLI Version: v0.6.0-alpha.1 (7678d67d)
```

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha2.

**Comment by StrongMonkey** [comment]
@rancher-max Also make sure `rio run --it ${image}` also works. This by default will launch the container and attach to its main process.

**Comment by rancher-max** [comment]
Fixed in v0.6.0-alpha.2.
Also verified `rio run -n attach2 -it izaac/attachtest:v1` and working as expected.

**Comment by izaac** [comment]
This is working for me too in rio `0.6.0-alpha.2`, I've tested in k3s 0.10.1 and EKS clusters.
